Sources/Workspace/Workspace+Configuration.swift

```diff
@@ -684,9 +684,13 @@ extension Workspace.Configuration {
                 return RegistryConfiguration()
             }
 
-            let data: Data = try fileSystem.readFileContents(self.path)
-            let decoder = JSONDecoder.makeWithDefaults()
-            return try decoder.decode(RegistryConfiguration.self, from: data)
+            do {
+                let data: Data = try fileSystem.readFileContents(self.path)
+                let decoder = JSONDecoder.makeWithDefaults()
+                return try decoder.decode(RegistryConfiguration.self, from: data)
+            } catch {
+                throw StringError("Failed loading registries configuration from '\(self.path)': \(error)")
```

Comment:
nit: in other places we use backquotes, not sure which of these quotes do we want to apply everywhere for consistency?

```suggestion
                throw StringError("Failed loading registries configuration from `\(self.path)`: \(error)")
```